resources/developer-guides/wallet-rpc.md

```diff
@@ -2564,6 +2565,37 @@ $ curl http://localhost:18082/json_rpc -d '{"jsonrpc":"2.0","id":"0","method":"c
 ```
 
 
+### **generate_from_keys**
+
+Restores a wallet from a given wallet address, view key, and optional spend key.
+
+Inputs:
+* *restore_height* - integer; (Optional; default = 0) The block height to restore the wallet from.
+* *filename* - string; The wallet's file name on the RPC server. 
+* *address* - string; The wallet's primary address.
+* *spendkey* - string; (Optional; omit to create a view-only wallet) The wallet's private spend key. 
+* *viewkey* - string; The wallet's private view key.
+* *password* - string; Sets the password for opening the wallet on the current file system.
```

Comment:
It's the password for opening the wallet on the monero-wallet-rpc server instead of the current file system.  "Password of the wallet." should be sufficient and is consistent with other entries.